circle.yml

```diff
@@ -23,7 +23,7 @@ dependencies:
 deployment:
   production:
     branch:
-      master
+      sunset
```

Comment:
This is needed to deploy to staging directly from this branch, instead of from the master.